foxy/distribution.yaml

```diff
@@ -2997,10 +2997,11 @@ repositories:
       - mavros
       - mavros_extras
       - mavros_msgs
+      - test_mavros
```

Comment:
This is failing linting due to the `test_mavros` package having a COLCON_IGNORE file in it, I believe.  Did you intend to release the test package, or can it be removed from the release?

Reply:
That's a bit unclear to me. Test package provides some useful examples, but it is not ported to ros2 yet. That's why it should be skipped...

Reply:
If it should be skipped, then it should not appear in this list.

Reply:
You can also place a `foxy.ignored` file in your release repository to have `bloom` automatically skip the package when releasing.  See https://github.com/ros2-gbp/geometry2-release for an example.